xp-archive/server/src/routing/site.ts

```diff
@@ -61,4 +61,21 @@ const setupBrowserRoutes = async (router: Router, htmlRenderer: HtmlRenderer) =>
 
         return res.send(content.html);
     });
+
+    router.get('/:contentId/:locale', async (req, res, next) => {
+        const { contentId, locale } = req.params;
+
+        const content = await contentService.fetchContent(contentId, locale);
+        if (!content) {
+            return next();
+        }
+
+        const contextContent = {
+            ...content,
+            html: undefined,
```

Comment:
Hvorfor settes html til undefined her?

Reply:
Fordi når jeg ikke setter den undefined rendres html-en dobbelt 
![image](https://github.com/user-attachments/assets/93cb26e9-283b-4c74-bb6a-4ffcfe033d26)

🤖🙂:
"The HTML is being set to undefined here because:

1. This route is handling the server-side rendering (SSR) of content
2. The raw HTML content is not needed in the context object since:
   - The HTML will be rendered by the htmlRenderer function
   - Including the raw HTML in the context would be redundant and potentially wasteful
   - The HTML content might be large and there's no need to pass it through the SSR context when it won't be used

This is a common pattern in SSR setups where you want to pass data to the renderer but exclude certain fields that aren't needed for the rendering process or that will be handled separately."